packages/jsdoc-plugins/lib/validator/types.js

```diff
@@ -35,6 +35,7 @@ const BASIC_TYPES = [
 	'ClientRect',
 
 	// Web APIs
```

Comment:
I would say that it implements some DOM interfaces, but not belongs to, as it contains huge amount of other things